gnovm/stdlibs/std/context.go

```diff
@@ -17,4 +18,6 @@ type ExecContext struct {
 	OrigSend      std.Coins
 	OrigSendSpent *std.Coins // mutable
 	Banker        BankerInterface
+	EventLogger   *sdk.EventLogger
+	EmittedEvents   abci.EventString
```

Comment:
What's the point of this field? And why is it singular (if I can emit multiple events in a single tx?)

Reply:
`EmittedEvents` fields stores the marshalled event data into a single `EventString` type. Due to the Events is a slice type therefore, EventString stored as a array-like JSON. So, I think it can be possible to emit multiple events in a single tx.

Reply:
Where do we use this field anyways?

Reply:
Removed as part of 
https://github.com/gnolang/gno/commit/7e391a90fd7a58fa2977df725cfd56a24f2b3f77